### ISSUE TITLE
No voice ?

### ISSUE THREAD

**Issue by Naivn3m** [maxbbraun/mirror#15]
Is there a Speech recognition feature ? If yes, then how to use it ?

**Comment by brandonmowat** [comment]
I don't believe there is. It's sophisticated, but I don't think it supports queries...

What kind of things would you like to ask/get information on?

**Comment by Naivn3m** [comment]
I want it to set a reminder for me!
Also to remind me of the pre-set reminder.
Show me the location on maps.
Open a particular activity through my voice.
Set an Alarm for me.
A Selfie command
Also detection of my presence, through some RFID tags.
Something related to health.
Read the news/feeds for me.

PS: I Know I'm asking for a lot, but I thought these things also can be implemented

**Comment by FlavourDaveMasterDDissle-theDtotheissle** [comment]
`but I thought these things also can be implemented` - so do it 😀

**Comment by brandonmowat** [comment]
yeah, you're always welcome to create a fork of your own or pull request. 

"Be the change you want to see" - some guy

**Comment by Naivn3m** [comment]
Sure! :P

But, I'm a newbie here

**Comment by brandonmowat** [comment]
@Naivn3m this would be a great learning experience

**Comment by Naivn3m** [comment]
@brandonmowat Sure! Thanks for helping.
